cdap-integration-test/src/main/java/co/cask/cdap/test/IntegrationTestBase.java

```diff
@@ -235,6 +235,11 @@ protected ClientConfig getClientConfig() {
       builder.setAccessToken(accessToken);
     }
 
+    String verifySSL = System.getProperty("verifySSL");
+    if (verifySSL != null) {
+      builder.setVerifySSLCert(Boolean.valueOf(verifySSL));
+    }
```

Comment:
If we change the default in `ClientConfig`, we'll likely forget to change it here too.